js/components/frame.js

```diff
@@ -25,16 +25,14 @@ class Frame extends ImmutableComponent {
 
   updateWebview () {
     let src = this.props.frame.get('src')
-    const isAboutURL = isSourceAboutUrl(src)
-    if (isAboutURL) {
-      src = getTargetAboutUrl(src)
-    }
     let contentScripts = ['content/webviewPreload.js']
-    if (this.props.frame.get('location') === 'about:preferences') {
+    // NOTE: use src instead of location here because location is sometimes
+    // not updated when redirecting to about:certerror
+    if (src === 'about:preferences' || src === 'about:certerror') {
```

Comment:
Noting that because of this change if we ever link to a page within an about: page which replaces the about page (i.e. not target=_blank) then that page will have elevated privs.  Seems a bit scary.  
Likewise if a content page links to about:preferences or about:certerror then the navigated frame won't work.